src/SigSpec.CodeGeneration.TypeScript/SigSpec.CodeGeneration.TypeScript.csproj

```diff
@@ -1,9 +1,22 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
+  <!-- Nuget-related properties -->
+  <PropertyGroup>
```

Comment:
Is this group needed?

Reply:
These props control the Nuget package creation. I wanted to put them into a separate group, so they are easier to find and modify when needed.